docs/products/redis/get-started.rst

```diff
@@ -1,59 +1,38 @@
-Get Started with Aiven for Redis
+Get started with Aiven for Redis
 ================================
 
-Get up and running with Aiven for Redis. This article shows you how to set up a Redis service in Aiven and learn more about the platform.
+Aiven for Redis services are managed from the `Aiven
+Console <https://console.aiven.io/>`__ .
 
-Create a new Aiven service
---------------------------
+You can start an Aiven for Redis service by following the steps laid out
+in :doc:`this article </docs/platform/howto/create_new_service>`.
 
-To create a new Aiven for Redis service:
+The view returns to the service list, where the new service is shown
```

Comment:
I feel it is a bit abrupt. You assume that the user followed the steps, but the instruction to follow the steps was a suggestion. I'd rephrase it. Something like this - 

Once the service creation is initiated, you can go to its dedicated service page .